### ISSUE TITLE
Small suggestions on Chapters 8-14: typos and more

### ISSUE THREAD

**Issue by jiapivialiu** [cmu-delphi/delphi-tooling-book#8]
Some overall suggestions:

- [x] R code and packages are highlighted in the `code` style, but some of them are left to be unhighlighted.
- [x] Some packages highlighted as `{code}`, but some are in `code`. I'm not sure if it's done for purpose. Otherwise, It might be better to unify them.
- [x] The word "data" is treated as a singular at many places throughout these chapters, but as far as I know it's a plural. Correct me if I'm wrong.
- [x] It'd be nice to link some key R functions to their vignettes, e.g., [`arx_forecaster()`](https://cmu-delphi.github.io/epipredict/reference/arx_forecaster.html), [`extract_frosting()`](https://cmu-delphi.github.io/epipredict/reference/extract_frosting.html).
- [x] The term "predictive interval" is used at multiple places, but "prediction interval" seems to be a more frequently used term.

Some thoughts on the overall layout:
- [x] The table of contents on the left side always appear for relatively small text sizes and is collapsed for large text sizes. But can we adjust the width of it arbitrarily?

[Chapter 8](https://cmu-delphi.github.io/delphi-tooling-book/epipredict.html): 
- [x] Miss a `(` [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/epipredict.qmd#L16).
- [x] When I was reading [Section 8.1](https://cmu-delphi.github.io/delphi-tooling-book/epipredict.html#baseline-models) the baseline models, I was wondering why these four models were suggested or used as baseline models. It'd be nice to have some brief explanation on these forecasters in this section regarding. For example, I'm wondering if they are alternatives to each other or if they are used for different research questions, and how to choose among four of them.
- [x] Also, I was confused at the beginning if these baseline models are models like linear regression, but then I realized that they were actually "frameworks" that allow various options of the models for model fitting. It'd be nice to clarify this somewhere in this section. Or it'd be clearer if the methodology of the forecasters are explained in more details.
- [x] The term "AR" is used without definition.
- [x] Section 8.2 mentions that “the baseline forecasters we provide requires post-processing”, but it sounds to me like the baseline forecasters wrap all four components (preprocessor -> postprocessor) from Section 8.1. It'd be better to clarify this in Section 8.1. 

[Chapter 9](https://cmu-delphi.github.io/delphi-tooling-book/forecast-framework.html)
- [x] There are grammar errors in this [sentence](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/forecast-framework.qmd#L188). It should be "recently available data" and "they lagged".
- [x] ~~This [sentence](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/forecast-framework.qmd#L194) mentions *production forecasting*, but the interpretation is not related to production forecasting. Is it a typo here? Is it actually *predictive forecasting*?~~

[Chapter 10](https://cmu-delphi.github.io/delphi-tooling-book/flatline-forecaster.html)
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/flatline-forecaster.qmd#L81), it does not quite make sense to me to use `$` operator after `epi_workflow`. @rachlobay 
- [x] In this [line](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/flatline-forecaster.qmd#L120), there is a redundant "the" between "order" and "that".

[Chapter 11](https://cmu-delphi.github.io/delphi-tooling-book/tidymodels-intro.html)
- [x] In [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L19), a typo: "parsniup" should be "parsnip" and a grammar mistake: "their" should be "its".
- [x] The "test" sets throughout this chapter look more like *validation* sets, which come from the same set of samples as training sets, but the actual *test* sets should come from a future release. Maybe clarify this in this [section](https://cmu-delphi.github.io/delphi-tooling-book/tidymodels-intro.html#data-sampling)?
- [x] In this [line](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L115), in the sentence of "In a recipe, ...": remove the redundant "the"; the estimation ... and the application ... "are" done automatically, "spare".
- [x] This is a question. Why do we need `-all_outcomes()` in this [line](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L124)? @rachlobay 
- [x] What does the operations section [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L130) refer to?
- [x] The last sentence [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L138). I don't believe it's not simple since to switch between the two packages we just need to change the name of argument from one to another. 
- [x] In this [line](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L140), remove "use" before "perform"; "a" single interface.
- [x] Add the chapter link in this [line](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L142).
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L156), for "simplicity".
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L164), "become" easier.
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L176), a redundant "the" between "prepare" and "recipe".
- [x] About the three bullets ([here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L158)). I'm confused with a) how they are realized in the following code? Is it through `add_model` and `add_recipe`, b) the third bullet point that "use the recipe on the predictor set to get the test set", which sounds like a future step after model fitting, and c) the pairwise difference between training set, finalized predictor set, predictor set, and test set. @rachlobay 
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L188): remove the redundant "that", add "to" before "replace".
- [x] There is a redundant "shown" [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L230).
- [x] A redundant "is" between "package" and "helps" [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L234).
- [x] We need a better way to elucidate the advantage of having the pre-processing steps [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-intro.qmd#L115). @rachlobay 

[Chapter 12](https://cmu-delphi.github.io/delphi-tooling-book/tidymodels-regression.html)
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L99) and [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L105), typos in "parsnip".
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L107)~~, I believe it's prediction interval, rather than confidence interval, since they are intervals for predicted values.~~
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L113), there is a redundant "the" before "each of the observed ...". 
- [x] ~~This [paragraph](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L177) seems to be quite long. Maybe highlight the key words for a better readability. (working on it in other branch)~~
- [x] A typo [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L179): it should be "one" not "once".
- [x] I'm confused with the first sentence [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/tidymodels-regression.qmd#L253). There may be some redundancies or it may lack a comma or something.

[Chapter 13](https://cmu-delphi.github.io/delphi-tooling-book/preprocessing-and-models.html)
- [x] I'm not sure it's proper to make all states as dummy variables [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/preprocessing-and-models.qmd#L99). We usually use (the number of states - 1) dummy variables to avoid collinearity. Is it the same case here?
- [x] ~~This chapter shows examples for panel data. It may worth mentioning it in the chapter name. (save for future work on chapter names)~~
- [x] ~~We may need a brief explanation of what kind of panel data we refer to and what they look like.~~

[Chapter 14](https://cmu-delphi.github.io/delphi-tooling-book/sliding-forecasters.html)
- [x] A redundant "in" after "such" [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/sliding-forecasters.qmd#L139).
- [x] ~~[Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/sliding-forecasters.qmd#L109) and [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/sliding-forecasters.qmd#L139), the confidence bands are more like prediction bands since they are for predicted values.~~
- [x] It seems like some comments are not commented successfully [here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/sliding-forecasters.qmd#L259).
- [x] [Here](https://github.com/cmu-delphi/delphi-tooling-book/blob/main/sliding-forecasters.qmd#L148), "those *models'* high variance".
